vendor/bat-native-confirmations/src/confirmations_impl.cc

```diff
@@ -50,12 +48,26 @@ ConfirmationsImpl::~ConfirmationsImpl() {
   StopPayingOutRedeemedTokens();
 }
 
+void ConfirmationsImpl::Initialize() {
+  BLOG(INFO) << "Initializing Confirmations";
```

Comment:
can we move this into vlog

Reply:
Sure right now we do not have VLOG in Confirmations but as soon as merged with Core this is planned

Reply:
great